packages/@react-spectrum/labeledvalue/test/LabeledValue.test.js

```diff
@@ -275,6 +275,35 @@ describe('LabeledValue', function () {
     expect(staticField).toHaveTextContent('10 – 20');
   });
 
+  it('renders correctly with ReactElement value', function () {
+    let {getByTestId} = render(
+      <LabeledValue
+        data-testid="test-id"
+        label="Field label"
+        value={<a href="https://test.com">test</a>} />
+    );
+
+    let staticField = getByTestId('test-id');
+    expect(staticField).toBeInTheDocument();
+    expect(staticField).toHaveTextContent('test');
+    expect(
+      within(staticField).getByRole('link', {name: 'test'})
+    ).toBeInTheDocument();
+  });
+
+  it('throws when an editable value is provided', async function () {
+    jest.spyOn(console, 'error').mockImplementation(() => {});
+    try {
+      render(
+        <LabeledValue
+          label="Field label"
+          value={<input />} />
+      );
+    } catch (e) {
+      expect(e.message).toEqual('LabeledValue cannot contain an editable value.');
```

Comment:
I think it's possible that this isn't throwing, and the test is still passing. We'll want to do the assertion outside of the catch to double check this. I'll try to double check this test when I have a chance. The change looks great, but there may be something we're missing in terms of the test setup. Sorry for the delays!